data/serializer/src/main/java/tech/pegasys/teku/api/schema/altair/BeaconStateAltair.java

```diff
@@ -40,82 +40,89 @@
-      @JsonProperty("current_epoch_participation") final byte[] current_epoch_participation,
-      @JsonProperty("justification_bits") final SszBitvector justification_bits,
-      @JsonProperty("previous_justified_checkpoint") final Checkpoint previous_justified_checkpoint,
-      @JsonProperty("current_justified_checkpoint") final Checkpoint current_justified_checkpoint,
-      @JsonProperty("finalized_checkpoint") final Checkpoint finalized_checkpoint,
-      @JsonProperty("inactivity_scores") final List<UInt64> inactivity_scores,
-      @JsonProperty("current_sync_committee") final SyncCommittee current_sync_committee,
-      @JsonProperty("next_sync_committee") final SyncCommittee next_sync_committee) {
+      @JsonProperty("previous_epoch_participation") final byte[] previousEpochParticipation,
+      @JsonProperty("current_epoch_participation") final byte[] currentEpochParticipation,
+      @JsonProperty("justification_bits") final SszBitvector justificationBits,
+      @JsonProperty("previous_justified_checkpoint") final Checkpoint previousJustifiedCheckpoint,
+      @JsonProperty("current_justified_checkpoint") final Checkpoint currentJustifiedCheckpoint,
+      @JsonProperty("finalized_checkpoint") final Checkpoint finalizedCheckpoint,
+      @JsonProperty("inactivity_scores") final List<UInt64> inactivityScores,
+      @JsonProperty("current_sync_committee") final SyncCommittee currentSyncCommittee,
+      @JsonProperty("next_sync_committee") final SyncCommittee nextSyncCommittee) {
     super(
-        genesis_time,
-        genesis_validators_root,
+        genesisTime,
+        genesisValidatorsRoot,
         slot,
         fork,
-        latest_block_header,
-        block_roots,
-        state_roots,
-        historical_roots,
-        eth1_data,
-        eth1_data_votes,
-        eth1_deposit_index,
+        latestBlockHeader,
+        blockRoots,
+        stateRoots,
+        historicalRoots,
+        eth1Data,
+        eth1DataVotes,
+        eth1DepositIndex,
         validators,
         balances,
-        randao_mixes,
+        randaoMixes,
         slashings,
-        justification_bits,
-        previous_justified_checkpoint,
-        current_justified_checkpoint,
-        finalized_checkpoint);
-    this.previous_epoch_participation = previous_epoch_participation;
-    this.current_epoch_participation = current_epoch_participation;
-    this.inactivity_scores = inactivity_scores;
-    this.current_sync_committee = current_sync_committee;
-    this.next_sync_committee = next_sync_committee;
+        justificationBits,
+        previousJustifiedCheckpoint,
+        currentJustifiedCheckpoint,
+        finalizedCheckpoint);
+    this.previousEpochParticipation = previousEpochParticipation;
+    this.currentEpochParticipation = currentEpochParticipation;
+    this.inactivityScores = inactivityScores;
+    this.currentSyncCommittee = currentSyncCommittee;
+    this.nextSyncCommittee = nextSyncCommittee;
```

Comment:
kind of a repeat, but basically all these fields are now incorrect in the api returns, which is why I'm suggesting putting their names back unless we have a good reason.